apps/warp/warp.talon

```diff
@@ -0,0 +1,10 @@
+os: mac
+app: warp
```

Comment:
You'll want to add an explicit match in core/application_matches.py. See [here](https://talon.wiki/Customization/Talon%20Framework/apps#non-explicit-app-header-matching) for why it's working without an explicit match.

Reply:
Thanks for the pointer. The example had it in it's own python file instead of in the application_matches.py. Let me know if I need to move this there intstead

Reply:
No, I think it's fine where it is… we should really get `core/application_matches.py` split out into multiple files rather than vice versa.